PlaybackLab/ControlsLockScreenSampleApp/ControlsLockScreenSampleApp/Views/ListTableViewController.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  ListTableViewController.swift
+//  ControlsLockScreenSampleApp
+//
+//  Created by Carlos Ceja on 10/18/17.
+//  Copyright © 2017 Ooyala. All rights reserved.
+//
+
+import UIKit
+
+class ListTableViewController: UITableViewController {
+  
+  private let options: [PlayerSelectionOption] = OptionDataSource.options()
+  
+  private var cellIdentifier: String = "cellOption"
+  
+  override func viewDidLoad() {
+    super.viewDidLoad()
+    tableView.tableFooterView = UIView()
```

Comment:
nice trick! I see it's to remove the visibility of empty rows 👍

Reply:
Yep, I don't like to see empty rows.